src/ngraph/frontend/onnx_import/op/conv_transpose.cpp

```diff
@@ -158,73 +79,31 @@ namespace ngraph
                         << "provided group attribute value must be a multiple of filter channels "
                            "count.";
 
-                    Shape data_batch_shape(data_shape.size(), 1);
-                    data_batch_shape.at(0) = data_shape.at(0);
-                    data_batch_shape.at(1) = weights_shape.at(1);
-
+                    std::shared_ptr<ngraph::Node> conv_node;
                     if (!output_shape.empty())
                     {
-                        if (output_shape.size() > num_spatial_dims)
-                        {
-                            output_shape.erase(std::begin(output_shape),
-                                               std::begin(output_shape) + 2);
-                        }
-                        for (int i = 0; i < num_spatial_dims; ++i)
-                        {
-                            padding_below[i] = strides[i] * (data_shape[i + 2] - 1) +
-                                               dilations[i] * (weights_shape[i + 2] - 1) -
-                                               data_dilation_strides[i] *
-                                                   (output_shape[i] - output_padding[i] - 1);
-                            if (padding_below[i] < 0)
-                            {
-                                // (int) -9 / 2 = -5 but we need -4
-                                // (int) -9 --> 9 / 2 = 4 --> -4
-                                padding_below[i] = -(-padding_below[i] / 2);
-                            }
-                            else
-                            {
-                                padding_below[i] /= 2;
-                            }
-                            padding_above[i] = padding_below[i];
-                            data_batch_shape[i + 2] = output_shape[i];
-                        }
+                        conv_node = std::make_shared<ngraph::op::GroupConvolutionTranspose>(
+                            data,
+                            filters,
+                            strides,
+                            dilations,
+                            output_padding,
+                            Shape(std::begin(output_shape), std::end(output_shape)),
```

Comment:
I think you can pass the output_shape directly in here

Reply:
Unfortunately not :(  
```
no known conversion for argument 6 from ‘std::vector<long int>’ to ‘const ngraph::Shape&’
```